managed/SavedSearchFundingCases.mgd.php

```diff
@@ -137,6 +138,13 @@
               'label' => E::ts('Recipient'),
               'sortable' => TRUE,
             ],
+            [
+              'type' => 'field',
+              'key' => 'application_process_progress',
+              'dataType' => 'Integer',
+              'label' => E::ts('Review Progress'),
+              'sortable' => TRUE,
```

Comment:
I think it would make sense to add `%` using the rewrite option `'rewrite' => '[application_process_progress] %'`.

Reply:
will change